packages/HtmlAgilityPack.1.4.9/HtmlAgilityPack.1.4.9.nuspec

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
-  <metadata>

```

Comment:
Any idea why this file is deleted?  I don't know if it's required, but it seems important?